data_structures/src/transaction_factory.rs

```diff
@@ -122,13 +116,14 @@ pub trait OutputsCollection {
 
     /// Generic inputs/outputs builder: can be used to build
     /// value transfer transactions and data request transactions.
+    #[allow(clippy::cast_sign_loss)]
+    #[allow(clippy::cast_possible_truncation)]
```

Comment:
I believe all the new allow clippy from this file can be removed now.
```suggestion
```

Reply:
Agreed and fixed!